### PR TITLE
feat: add total

### PR DESCRIPTION
**Pull request by KaczDev**
Hi,

Thanks for the script it's very beneficial!

I added the printout of the summed mem used.
It's helpful when fighting the mighty oom_killer.

### CODE REVIEW

File: vmrss [hunk 1]
```diff
@@ -27,7 +27,7 @@ while [ ${#arr[@]} -gt 0 ]; do
       | grep $GREP_OPTS -o '[0-9]\+' \
       | awk '{print $1/1024}')
     #Add decimals to total
-    total = $(echo $mem+$total | bc)
+    total=$(echo $mem+$total | bc)
```
Comment (cgsdev0) [inline vmrss:30]: we don't need to fork to accomplish this
```suggestion
    ((total+=mem))
```

Comment (KaczDev) [reply vmrss:30]: Thanks for the advice!
Does it work with decimals?
I tested it on my machine with bash 5 and 4.4 (on a VM) and got this error
```
./vmrss.sh: line 30: ((: total+=43863.9: syntax error: invalid arithmetic operator (error token is ".9")
```

**Comment by ThePrimeagen** [comment]
@cgsdev0 could you let me know when you think this is good to merge?

since your bash feedback is pretty much infinitely better than mine

**Comment by ThePrimeagen** [comment]
in fact @cgsdev0 just gave you collaborator privs.  if you just want to merge when you think good, by all means

**Comment by ThePrimeagen** [comment]
ok, i want this change now

i'll see if it works and just merge, @cgsdev0 please feel free to update in any way and just push to master, yolo, --force, --force-because-i-trust-you

**Comment by ThePrimeagen** [comment]
works on my machine

File: vmrss (hunk 1 above)
Comment (cgsdev0) [reply vmrss:30]: oh you're right it doesn't work with decimals rip

**Pull request merged**